python/pyspark/sql/connect/session.py

```diff
@@ -706,9 +724,9 @@ def createDataFrame(
         else:
             local_relation = LocalRelation(_table)
 
-        cache_threshold = self._client.get_configs("spark.sql.session.localRelationCacheThreshold")
+        cache_threshold = conf_getter["spark.sql.session.localRelationCacheThreshold"]
```

Comment:
Shall we just get all the confs in batch eagerly? Seems like we should get the conf once anyway.

Reply:
there are 2 cases that we don't need the configs:
1, the local data is empty, and the schema is specified, it returns a valid empty df;
2, the creation fails due to some assertions